feature/experimental/bgp/otg_tests/base_bgp_session_parameters/base_bgp_session_parameters_test.go

```diff
@@ -0,0 +1,525 @@
+	fptest.LogQuery(t, "DUT BGP Config", dutConfPath.Config(), gnmi.GetConfig(t, dut, dutConfPath.Config()))
+
+	// ATE Configuration.
+	t.Log("Configure port and BGP configs on ATE")
+	ate := ondatra.ATE(t, "ate")
+	topo := configureATE(t, &bgpTestParams{localAS: ateAS, peerIP: dutAttrs.IPv4}, connExternal, noAuth)
+
+	t.Log("Pushing config to ATE and starting protocols...")
+	otg := ate.OTG()
+	otg.PushConfig(t, topo)
+	otg.StartProtocols(t)
+
+	// Verify Port Status
+	t.Log("Verifying port status")
+	verifyPortsUp(t, dut.Device)
+
+	// Verify BGP status
+	t.Log("Check BGP parameters")
+	verifyBgpTelemetry(t, dut)
+
+	// Verify BGP capabilities
+	t.Log("Check BGP Capabilities")
+	verifyBGPCapabilities(t, dut)
+
+	// Send Cease Notification from ATE to DUT
+	t.Log("Send Cease Notification from OTG to DUT -- ")
+	otg.SetControlAction(t, createCeaseAction(t))
+
+	// Verify BGP session state : ACTIVE
+	t.Log("Verify BGP session state : ACTIVE")
+	gnmi.Await(t, dut, nbrPath.SessionState().State(), time.Second*60, oc.Bgp_Neighbor_SessionState_ACTIVE)
+
+	// Verify if Cease notification is received on DUT.
+	t.Log("Verify Error code received on DUT: BgpTypes_BGP_ERROR_CODE_CEASE")
+	code := gnmi.Get(t, dut, nbrPath.Messages().Received().LastNotificationErrorCode().State())
+	if code != oc.BgpTypes_BGP_ERROR_CODE_CEASE {
+		t.Errorf("On disconnect: expected error code %v, got %v", oc.BgpTypes_BGP_ERROR_CODE_CEASE, code)
+	}
+
+	// Clear config on DUT and ATE
+	otg.StopProtocols(t)
+	bgpClearConfig(t, dut)
+}
+
+// TestPassword is to verify md5 authentication password on DUT.
+// Verification is done through BGP adjacency implicitly.
+func TestPassword(t *testing.T) {
+	// DUT configurations.
+	t.Log("Start DUT config load:")
+	dut := ondatra.DUT(t, "dut")
+
+	// Configure interface on the DUT
+	t.Log("Start DUT interface Config")
+	configureDUT(t, dut)
+
+	// Configure Network instance type on DUT
+	t.Log("Configure Network Instance")
+	dutConfNIPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut))
+	gnmi.Replace(t, dut, dutConfNIPath.Type().Config(), oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+
+	t.Log("Configure BGP")
+	dutConfPath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	statePath := gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP").Bgp()
+	nbrPath := statePath.Neighbor(ateAttrs.IPv4)
+
+	bgpClearConfig(t, dut)
+	dutConf := bgpCreateNbr(&bgpTestParams{localAS: dutAS, peerAS: ateAS}, dut)
+	gnmi.Replace(t, dut, dutConfPath.Config(), dutConf)
+	t.Log("Configure matching Md5 auth password on DUT")
+	gnmi.Replace(t, dut, dutConfPath.Bgp().Neighbor(ateAttrs.IPv4).AuthPassword().Config(), authPassword)
+
+	fptest.LogQuery(t, "DUT BGP Config", dutConfPath.Config(), gnmi.GetConfig(t, dut, dutConfPath.Config()))
```

Comment:
GetConfig has been deprecated in favor of Get.

Reply:
fixed